scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -551,9 +552,16 @@ public void fetchMetadataOfPreciousResults() {
 
                                                             @Override
                                                             public void onFailure(Throwable caught) {
-                                                                LogModel.getInstance()
-                                                                        .logCriticalMessage("Error while fetching metadata of precious results:\n" +
-                                                                                            JSONUtils.getJsonErrorMessage(caught));
+                                                                String msg = JSONUtils.getJsonErrorMessage(caught);
+                                                                if (msg.equals("HTTP 403 Forbidden")) {
```

Comment:
Maybe you could replace `msg.equals` by `msg.contains` it would be more resilient if the message is updated.

Reply:
Copy that